_docs/technical/protocols/2017-01-13-binary.md

```diff
@@ -559,48 +573,182 @@ Example:
 ### Address Attributes
 
 ``` haskell
-newtype AddrPkAttrs = AddrPkAttrs
-    { addrPkDerivationPath :: Maybe HDAddressPayload
-    } deriving (Eq, Ord, Show, Generic, Typeable, NFData)
+-- | Additional information stored along with address. It's intended
+-- to be put into 'Attributes' data type to make it extensible with
+-- softfork.
+data AddrAttributes = AddrAttributes
+    { aaPkDerivationPath  :: !(Maybe HDAddressPayload)
+    , aaStakeDistribution :: !AddrStakeDistribution
+    } deriving (Eq, Ord, Show, Generic, Typeable)
+```
+
+[Source code
+link](https://github.com/input-output-hk/cardano-sl/blob/524bb4d6d6620bec7e269e33de1154d2b1be5183/core/Pos/Core/Types.hs#L199)
 
+Address attributes hold additional information about address:
+
+- encrypted derivation path from the root of HD wallet tree (`aaPkDerivationPath`)
+- stake distribution associated with this address (`aaStakeDistribution`)
+
+Note that address may not contain an encrypted derivation path. Such addresses
+are are usually derived from level 0 (wallet) or level 1 (account) public keys
```

Comment:
Typo
> are are

Also, it's not clear at first whether do you mean addresses w/o or w/ derivation path